cli/tests/integrations/test_package.py

```diff
@@ -75,59 +76,16 @@ def _chronos_description(app_ids):
 
 
 def test_package():
-    stdout = b"""Install and manage DCOS software packages
-
-Usage:
-    dcos package --config-schema
-    dcos package --info
-    dcos package describe [--app --options=<file> --cli] <package_name>
-    dcos package install [--cli | [--app --app-id=<app_id>]]
-                         [--options=<file> --yes] <package_name>
-    dcos package list [--json --endpoints --app-id=<app-id> <package_name>]
-    dcos package search [--json <query>]
-    dcos package sources
-    dcos package uninstall [--cli | [--app --app-id=<app-id> --all]]
-                 <package_name>
-    dcos package update [--validate]
-
-Options:
-    -h, --help         Show this screen
-    --info             Show a short description of this subcommand
-    --version          Show version
-    --yes              Assume "yes" is the answer to all prompts and run
-                       non-interactively
-    --all              Apply the operation to all matching packages
-    --app              Apply the operation only to the package's application
-    --app-id=<app-id>  The application id
-    --cli              Apply the operation only to the package's CLI
-    --options=<file>   Path to a JSON file containing package installation
-                       options
-    --validate         Validate package content when updating sources
-
-Configuration:
-    [package]
-    # Path to the local package cache.
-    cache_dir = "/var/dcos/cache"
-
-    # List of package sources, in search order.
-    #
-    # Three protocols are supported:
-    #   - Local file
-    #   - HTTPS
-    #   - Git
-    sources = [
-      "file:///Users/me/test-registry",
-      "https://my.org/registry",
-      "git://github.com/mesosphere/universe.git"
-    ]
-"""
+    stdout = pkg_resources.resource_string(
```

Comment:
I think we should keep these decoupled.  Otherwise, we're not testing that `--help` hasn't changed unexpectedly.